db/schema.rb

```diff
@@ -12,7 +12,25 @@
 
 ActiveRecord::Schema.define(version: 2018_08_17_100339) do
 
-  create_table "list_items", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
+  create_table "link_check_reports", options: "ENGINE=InnoDB DEFAULT CHARSET=latin1", force: :cascade do |t|
```

Comment:
@jonodrew Do we ever use this table?

Reply:
No, I thought I'd removed it in an earlier commit. Not sure how that's sneaked back in there.

Reply:
I'll get rid of it and fix-up.